src/Pages/Home/component/All.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import AllCard from './Card/AllCard';
+import '../Home.scss';
+
+class All extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      allList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://10.58.2.238:8000/products?category=전체', {
```

Comment:
- API 부는 config.js 파일을 만들어서 관리해주세요!
```suggestion
    fetch(`${GET_PRODUCTS_API}?category=전체`, {
```

Reply:
넵 Config.js 만들어서 관리하고 있습니다!